docs/website/docs/command-reference/logs.md

```diff
@@ -11,7 +11,7 @@ If you haven't already done so, you must [initialize](../command-reference/init)
 init` command. 
 
 ```shell
-odo logs [--follow] [--dev | --deploy]
+odo logs [--follow] [--dev | --deploy] [--platform podman]
```

Comment:
Maybe it's just me, but I read this as if the only value I can pass to `--platform` is `podman`.
How about this?

```suggestion
odo logs [--follow] [--dev | --deploy] [--platform {cluster|podman}]
```